### ISSUE TITLE
Magento 2.3 Unable to Open Admin Panel for Design Configuration to Upload Logo

### ISSUE THREAD

**Issue by jurpro** [magento/magento2#19486]
After upgrade from Magento 2.2.7 to Magento 2.3.0, I unable open Global Design Configuration.
### Preconditions (*)
I was upgraded my site from Magento Community Edition 2.2.7 to 2.3.0. It was succesfull and fine. But unfortunately, cannot open Global Design Configuration.

### Steps to reproduce (*)
1. Open Admin Panel.
2. Chose "Content"
3. Chose "Design" ==> "Configuration"
4. After Design Configuration page loaded, then choose "Global" and click "Edit"

### Expected result (*)
Ussually on Global Design Configuration, we can upload Logo, email logo, and icon for homepage.

### Actual result (*)
It is freeze nothing happened. Only loading never ending. Even you refresh the page cannot succesfully loaded to show you some dialog for header, email or icon logo. Like this screenshot.
![error logo](https://user-images.githubusercontent.com/5201127/49304664-a19c1a00-f4ff-11e8-8d67-6a08d615d167.jpg)

**Comment by magento-engcom-team** [comment]
Hi @jurpro. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@jurpro do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by mahesh-rajawat** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @maheshWebkul721. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @maheshWebkul721, here is your [Magento instance](https://i-19486-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-19486-2-3-0.instances.magento-community.engineering/admin](https://i-19486-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @maheshWebkul721. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by mahesh-rajawat** [comment]
@jurpro issue can not reproduce on 2.3.0 instance.

**Comment by jurpro** [comment]
what do you mean? why this is closed without answer? I cannot open the Global Design Configuration to upload logo etc and then you closed this issue? what....

**Comment by mahesh-rajawat** [comment]
@jurpro i am reopening this issue, please check the same issue in 2.3-develop instance.  You can ask @magento-engcom-team to provide you the Magento 2.3 instance.

**Comment by jurpro** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @jurpro. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @jurpro, here is your [Magento instance](https://i-19486-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-19486-2-3-0.instances.magento-community.engineering/admin](https://i-19486-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by jurpro** [comment]
it seems theme bug. I will tell to my dev theme. Thanks

**Comment by mahesh-rajawat** [comment]
@jurpro Thanks to cross check it. I am closing this issue now.

**Comment by andy17612** [comment]
@jurpro Are you sure is theme bug? i am getting same issue,thank you

**Comment by jurpro** [comment]
@andy17612 I don't know. My dev theme still help without solved. They told me it is very strange because they didn't change any core files.
If you got the same error, then this is a bug from 2.3.0 for migrating from 2.2.7

I don't understand what and how to use Magento vanilla. Ussually if I ask in github, the solution is from team or other users not using vanilla.

**Comment by andy17612** [comment]
@jurpro @magento-engcom-team Main Website and Main Website Store are working, just Global are not working.
This is Global page message.
> file-uploader.min.js:1 Uncaught ReferenceError: Base64 is not defined
>     at UiClass.processFile (file-uploader.min.js:1)
>     at Array.map (<anonymous>)
>     at UiClass.setInitialValue (file-uploader.min.js:1)
>     at UiClass.initialize (abstract.min.js:1)
>     at UiClass.initialize (wrapper.min.js:3)
>     at new UiClass (class.min.js:3)
>     at Object.initComponent (layout.min.js:8)
>     at fire (jquery.min.js:212)
>     at Object.fireWith [as resolveWith] (jquery.min.js:218)
>     at Object.deferred.(/index.php/admin/theme/design_config/edit/scope/default/key/3d098cdba6b78b5b34afeb20d3e1f3d052972a2b3b29bc3d63761/anonymous function) (http://www.domain.com/pub/static/version154399332/adminhtml/Magento/backend/en_US/jquery.min.js:220:56)

**Comment by andy17612** [comment]
@jurpro https://github.com/magento/magento2/issues/19549

**Comment by michelecanal** [comment]
same problem

**Comment by simonmaass** [comment]
i am having the same issue after upgrading from 2.2.7 to 2.3

**Comment by steveb2050** [comment]
Ditto, i am having the same issue after upgrading from 2.2.7 to 2.3.  

Step...
Content => Design => Configuration
click 'Edit' to access config settings for registered theme
page tries to load.. https://my...site/magento2/admin...location/theme/design_config/edit/scope/stores/scope_id/1/

Page gets hung and doesn't load - spinning graphic. 

Repeats in all three browers - GC, Firefox and IE

Anyone have any idea what is going on here and how to fix this. Can't change logo, html headers, etc.

**Comment by steveb2050** [comment]
Updating last comment - the solution that worked for me... 

See issue [19549](https://github.com/magento/magento2/issues/19549) - which is now closed

And the solution by @SoCix  (thanks!)

For completeness, this was also addressed on stackexchange => [here](https://magento.stackexchange.com/questions/252416/magento-2-3-design-configuration-global-page-keep-loading)
